src/lib/Microsoft.Health.Fhir.Ingest/Data/MeasurementToEventMessageAsyncCollector.cs

```diff
@@ -4,25 +4,33 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using Azure.Messaging.EventHubs;
 using EnsureThat;
-using Microsoft.Azure.WebJobs;
 using Microsoft.Health.Fhir.Ingest.Service;
 using Newtonsoft.Json;
 
 namespace Microsoft.Health.Fhir.Ingest.Data
 {
     public class MeasurementToEventMessageAsyncCollector :
-        IAsyncCollector<IMeasurement>
```

Comment:
>IAsyncCollector<IMeasurement> [](http://example.com/codeflow?start=8&length=29)

Can we keep the implemenation for IAsyncCollector as well?  That will allow you use the old signature and logic for Azure Function implemenation based on my prior comment there with out needing to retest.  Basically add this enchancement for the new version but keep the old impl for backwards compat.

Reply:
I actually think this breaks the Azure Function.  The function has the binding   [EventHubMeasurementCollector("output", Connection = "OutputEventHub")] IAsyncCollector<IMeasurement> output and this change breaks it since it no longer conforms to this interface.

Reply:
`IEnumerableAsyncCollector ` implements `IAsyncColloector. But i'll look into this more to confirm.

Reply:
Nevermind, it looks like this is a very similar class but different from MeasurementToEventAsyncCollector so the Azure Function will still compile but I still feel it would be better to leave the Azure Function implementation unchanged.

Reply:
Ya I just realized they are different, but named very similar and were implementing the same interface hence my confusion.